app/app.py

```diff
@@ -84,18 +93,125 @@ def image_post():
    
 
 
-    
+  #------------------------------------------------------  
+class Register(FlaskForm):
+    username = StringField('username', validators=[InputRequired()])
+    email = EmailField('email', validators=[InputRequired(), Email()])
+    firstname = StringField('firstname', validators=[InputRequired()])
+    lastname = StringField('lastname', validators=[InputRequired()])
+    password = PasswordField('password', validators=[InputRequired(), Length(max=18,min=6)])
+    #submit = SubmitField('Submit')
+
+usernameinfo = ''
```

Comment:
```suggestion
usernameinfo = ''
firstnameinfo = ''
lastnameinfo = ''
emailinfo = ''
```
I think this was left in by accident, delete if not needed